src/cityreader/cityreader.py

```diff
@@ -16,13 +28,25 @@
 # should not be loaded into a City object.
 cities = []
 
+
 def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
+  # For each city record, create a new City instance and add it to the
   # `cities` list
-    
+
+    with open('cities.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile)
+        for row in reader:
+            # research the `next()` function in `csv.reader` to avoid using the conditional on each line in this loop.
+            if row[0] != 'city':
```

Comment:
You might think about getting in the habit of using python keywords for comparisons. They're a lot more powerful than the symbol versions. 
`and, or, not, is`